### ISSUE TITLE
.RNG.seed warnings when estimating the prior

### ISSUE THREAD

**Issue by PuzzledFace** [openpharma/rbqmR#6]
```
fitBayesPoissonModel(data=NULL)

<output deleted>
Warning messages:
1: In checkParameters(init.values[[i]]) :
  NULL initial value supplied for variable(s) .RNG.seed
2: In checkParameters(init.values[[i]]) :
  NULL initial value supplied for variable(s) .RNG.seed
3: In checkParameters(init.values[[i]]) :
  NULL initial value supplied for variable(s) .RNG.seed
4: In checkParameters(init.values[[i]]) :
  NULL initial value supplied for variable(s) .RNG.seed
```

But

```
cavalryDeaths %>% 
  group_by(Corps) %>% 
  summarise(Deaths=sum(Deaths), Years=sum(Year)) %>%
  fitBayesPoissonModel(Deaths, Years)
<output deleted>
```

Note no warnings.  Why?

This does not occur with `fitBayesBinomialModel`.

**Comment by PuzzledFace** [comment]
Possibly related:

The test written to demonstrate the closure of this issue now (Dec 2022) fails when run as part of `devtools::check()`:

```
Error : Error reading initial values for chain 1 (see output above for more details, and use failed.jags('inits') to see init file syntax with line numbers)
[ FAIL 1 | WARN 0 | SKIP 0 | PASS 209 ]

══ Failed tests ════════════════════════════════════════════════════════════════
── Failure (test-fitBayesPoissonModel.R:4:3): Issue #3 has been resolved ───────
fitBayesPoissonModel(data = NULL)$status == "OK" is not TRUE

`actual`:   FALSE
`expected`: TRUE 

[ FAIL 1 | WARN 0 | SKIP 0 | PASS 209 ]
Error: Test failures
Execution halted
```

But passes when the test file is run interactively:

```
══ Testing test-fitBayesPoissonModel.R ═════════════════════════════════════════

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 0 ]Loading required namespace: rjags

[ FAIL 0 | WARN 1 | SKIP 0 | PASS 0 ]
[ FAIL 0 | WARN 2 | SKIP 0 | PASS 0 ]
[ FAIL 0 | WARN 3 | SKIP 0 | PASS 0 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 0 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 1 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 2 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 3 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 4 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 5 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 6 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 7 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 8 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 9 ]
[ FAIL 0 | WARN 4 | SKIP 0 | PASS 10 ]

── Warning (test-fitBayesPoissonModel.R:4:3): Issue #3 has been resolved ───────
NULL initial value supplied for variable(s) .RNG.seed
Backtrace:
  1. testthat::expect_true(...)
       at test-fitBayesPoissonModel.R:4:2
  6. rbqmR::.autorunJagsAndCaptureOutput(...)
 14. runjags::autorun.jags(...)
 15. runjags::autoextend.jags(...)
 16. runjags::extend.jags(...)
 18. runjags:::as.jags.runjags(runjags.object, adapt = 0, quiet = silent.jags)
 26. rjags::jags.model(...)
 27. rjags checkParameters(init.values[[i]])

── Warning (test-fitBayesPoissonModel.R:4:3): Issue #3 has been resolved ───────
NULL initial value supplied for variable(s) .RNG.seed
Backtrace:
  1. testthat::expect_true(...)
       at test-fitBayesPoissonModel.R:4:2
  6. rbqmR::.autorunJagsAndCaptureOutput(...)
 14. runjags::autorun.jags(...)
 15. runjags::autoextend.jags(...)
 16. runjags::extend.jags(...)
 18. runjags:::as.jags.runjags(runjags.object, adapt = 0, quiet = silent.jags)
 26. rjags::jags.model(...)
 27. rjags checkParameters(init.values[[i]])

── Warning (test-fitBayesPoissonModel.R:4:3): Issue #3 has been resolved ───────
NULL initial value supplied for variable(s) .RNG.seed
Backtrace:
  1. testthat::expect_true(...)
       at test-fitBayesPoissonModel.R:4:2
  6. rbqmR::.autorunJagsAndCaptureOutput(...)
 14. runjags::autorun.jags(...)
 15. runjags::autoextend.jags(...)
 16. runjags::extend.jags(...)
 18. runjags:::as.jags.runjags(runjags.object, adapt = 0, quiet = silent.jags)
 26. rjags::jags.model(...)
 27. rjags checkParameters(init.values[[i]])

── Warning (test-fitBayesPoissonModel.R:4:3): Issue #3 has been resolved ───────
NULL initial value supplied for variable(s) .RNG.seed
Backtrace:
  1. testthat::expect_true(...)
       at test-fitBayesPoissonModel.R:4:2
  6. rbqmR::.autorunJagsAndCaptureOutput(...)
 14. runjags::autorun.jags(...)
 15. runjags::autoextend.jags(...)
 16. runjags::extend.jags(...)
 18. runjags:::as.jags.runjags(runjags.object, adapt = 0, quiet = silent.jags)
 26. rjags::jags.model(...)
 27. rjags checkParameters(init.values[[i]])


[ FAIL 0 | WARN 4 | SKIP 0 | PASS 10 ]

Test complete
```

Initially, in Nov 2022, the test passed.

**Comment by rbqmR-review** [comment]
Fixed as part of #11.
